examples/notebooks/Getting Started/Tutorial 2 - Compare models.ipynb

```diff
@@ -122,6 +122,42 @@
     "In this tutorial we have seen how easy it is to run and compare different electrochemical models. In [Tutorial 3](./Tutorial%203%20-%20Basic%20plotting.ipynb) we show how to create different plots using PyBaMM's built-in plotting capability."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## References"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The relevant papers to this notebook are:"
```

Comment:
I think "papers relevant to" or "relevant papers for" is more gramatically correct

Reply:
Done